### PR TITLE
Update Salt Formula path

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/7937

### CODE REVIEW

**Comment by mcalmer** [comment]
Well, our formulas are for 3.2 and 4.0. So if we change it new version will not work anymore on 3.2.
So I would wait until we are near EOL of 3.2 before changing it. At least until beginning of next year.

**Comment by renner** [comment]
Thanks @mcalmer, so we should probably only point people to the new directories **after** we actually changed the Formulas (~ beginning of 2020). In the meantime, could we maybe just mention the new directory where it makes sense and that it is supported for Formulas to also be used with standalone Salt?

File: modules/salt/pages/salt-formulas.adoc [hunk 1]
```diff
@@ -184,7 +190,7 @@ These files are detected as form recipes and may be applied to groups and system
 For example, the [path]``form.yml`` that is included with the [package]#locale-formula# is placed in:
 
 ----
-/usr/share/susemanager/formulas/metadata/locale/form.yml
+/usr/share/salt-formulas/metadata/locale/form.yml
```
Comment (renner) [inline modules/salt/pages/salt-formulas.adoc]: This example should reference the old directory.

File: modules/salt/pages/salt-formulas.adoc [hunk 2]
```diff
@@ -944,7 +950,7 @@ Then it will start the DNS service (``named``).
 
 With the dhcpd formula you set up and configure a DHCP server (Dynamic Host Configuration Protocol).
 For technical information about the dhcpd formula and low-level pillar data, see the Pillar example file
-[path]``/usr/share/susemanager/formulas/metadata/dhcpd/pillar.example``.
+[path]``/usr/share/salt-formulas/metadata/dhcpd/pillar.example``.
```
Comment (renner) [inline modules/salt/pages/salt-formulas.adoc]: The old directory is still valid for both examples.

File: modules/architecture/pages/salt-data-locations.adoc [hunk 3]
```diff
@@ -16,7 +16,7 @@ Global static data *should not* be customized or edited by {productname} users.
 
 | /usr/share/susemanager/salt/  | custom modules, states, grains
 | /usr/share/susemanager/pillar_data/ |	global pillar data
-| /usr/share/susemanager/formulas/ | formulas
+| /usr/share/salt-formulas/ | formulas
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply modules/architecture/pages/salt-data-locations.adoc]: Updated. Note, however, that this book is de-published for the moment anyway.

**Comment by Loquacity** [comment]
> LGTM, but I have not touched this in a while. I trust you :) I guess we are waiting on a review from renner.

Yep! @renner over to you ;)

**Comment by jcayouette** [comment]
This looks good to me. I will merge. We can resolve any issues later.

**Pull request merged**